app/styles/ui/check-runs/_ci-check-run-list-item.scss

```
@@ -12,10 +12,14 @@
   .ci-check-list-item-detail {
     flex: 1;
     margin: auto;
+    width: 80%;
```

I think we should avoid setting the size of elements to a % when it's not 100%, unless we know the value we use will work with any width of the container. For example, if we made the popover wider, would 80% still be a good value? Probably not since the check status icon and the "View On GitHub" icon have fixed widths. We want the titles to take the rest.

For me, this change worked (and I think it makes more sense):
```suggestion
    overflow-x: hidden;
```